cmd/queue/main.go

```diff
@@ -274,7 +285,7 @@ func main() {
 
 	timeoutHandler := queue.TimeToFirstByteTimeoutHandler(http.HandlerFunc(handler(reqChan, breaker, httpProxy, h2cProxy)),
 		time.Duration(revisionTimeoutSeconds)*time.Second, "request timeout")
-	composedHandler := pushRequestLogHandler(timeoutHandler)
+	composedHandler := pushRequestMetricHandler(pushRequestLogHandler(timeoutHandler))
```

Comment:
Did you mean to replace the log handler?

Reply:
No. Hmm, what I did is adding a handler layer to the log handler, isn't it?